README.md

```diff
@@ -69,7 +69,30 @@ source <(kompose completion bash)
 source <(kompose completion zsh)
 ```
 
-## Building
+## Development and building of Kompose
+
+
+### Downloading the latest (master) binary
+
+"Nightly" binaries are generated on a per-merge basis and uploaded to [Bintray](https://bintray.com/kompose)
+
+__Linux and macOS:__
+
+```sh
+# Linux 
+curl -L https://dl.bintray.com/kedgeproject/kedge/latest/kompose-linux-amd64 -o kompose
```

Comment:
link is not working :(

`https://dl.bintray.com/kedgeproject/kedge/latest/kompose-linux-amd64` looks weird